src/schema/v2/me/myCollection.ts

```diff
@@ -90,14 +97,13 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
 
     // We're setting the size to 100 here because we're going to filter and sort these artworks later manualy.
     // The idea is to fetch (almost) all artworks and then filter/sort them in memory.
-    const paginationArgs = options.sortByLastAuctionResultDate
-      ? { size: MAX_COLLECTION_SIZE }
+    const optionsAndPaginationArgs = options.sortByLastAuctionResultDate
+      ? { size: MAX_COLLECTION_SIZE, ...options }
       : convertConnectionArgsToGravityArgs(options)
 
     const gravityOptions = {
-      exclude_purchased_artworks: options.excludePurchasedArtworks,
       total_count: true,
-      ...paginationArgs,
+      ...snakeCaseKeys(optionsAndPaginationArgs),
```

Comment:
**Q:** shouldn't we still send `artist_ids` if it's undefined since we're only looking for its presence [here](https://github.com/artsy/gravity/pull/16558/files#r1268100822). Or are we stripping falsy values somewhere?

Reply:
If `artist_ids` isn't passed in the GraphQL query, we don't send it to Gravity either. I guess this is the expected behavior. The presence check will fail and the artworks won't be filtered by artistIDs.

Reply:
my bad, i was multi tasking. That's what I meant, it's being sent now even if it's `undefined` right?

Reply:
I tested it without `artistIds` and the param wasn't sent as `undefined`  to Gravity.